src/jetson/main.py

```diff
@@ -122,9 +140,67 @@ def detect(self,
                     kp_y = detections[i, 4 + k * 2 + 1] * img.shape[0]
 
                 bboxes.append((xmin, ymin, xmax, ymax))
+
             return bboxes
 
 
+        elif (self.model_name == 'retinaface'):
```

Comment:
if were gonna have different models we use like this I think we should revisit how this is implemented other than long if statements. its fine for now but this shouldn't be seen as a long term solution

Reply:
I agree. Please look into this when you get a chance @pawar4 @ZPBerg @jbagnara